reindexer/src/main/scala/uk/ac/wellcome/platform/reindexer/services/ReindexTargetService.scala

```diff
@@ -41,22 +41,24 @@ abstract class ReindexTargetService[T <: Reindexable[String]](
 
   protected val scanamoQueryStreamFunction: ScanamoQueryStreamFunction
 
-  private def updateVersion(resultGroup: List[ScanamoQueryResult]): Boolean = {
+  private def updateVersion(newReindexVersion: Int,
```

Comment:
I would use multiple parameters here `updateVersion(requestedVersion: Int)(resultGroup: List[ScanamoQueryResult])`, so it can be called later `scanamoQueryStreamFunction(
      scanamoQueryRequest,
      updateVersion(requestedVersion))`